lib/stream/tag.rb

```diff
@@ -5,46 +5,43 @@
 #   the COPYRIGHT file.
 
 class Stream::Tag < Stream::Base
-  attr_accessor :tag_name, :people_page , :people_per_page
+  attr_accessor :tag_names, :people_page, :people_per_page
 
-  def initialize(user, tag_name, opts={})
-    self.tag_name = tag_name
+  def initialize(user, tag_names, opts={})
+    self.tag_names = tag_names.downcase.gsub("#", "")
     self.people_page = opts[:page] || 1
     self.people_per_page = 15
     super(user, opts)
   end
 
-  def tag
-    @tag ||= ActsAsTaggableOn::Tag.named(tag_name).first
+  def tags
+    @tags ||= ActsAsTaggableOn::Tag.named_any(tag_names.split(" "))
   end
 
   def display_tag_name
-    @display_tag_name ||= "##{tag_name}"
+    @display_tag_name ||= tag_names.split(" ").map {|t| "##{t}" }.join(" ")
   end
 
   def tagged_people
-    @people ||= ::Person.profile_tagged_with(tag_name).paginate(:page => people_page, :per_page => people_per_page)
+    @tagged_people ||= ::Person.profile_tagged_with(tag_names).paginate(page: people_page, per_page: people_per_page)
```

Comment:
This doesn't really work with multiple tag names, is it expected?

The search results shows nothing, even if there is some profiles with multiple tags.

![image](https://user-images.githubusercontent.com/10187586/178121248-085b2e40-0be8-40f1-9f7d-cebaebe8fe8b.png)